doc/gui/index.rst

```diff
@@ -1,16 +1,37 @@
-=====
- GUI
-=====
+===
+GUI
+===
 
 The brand-new lightweight web-based hnn GUI that works with hnn-core.
 
+.. raw:: html
 
-Launch GUI in a Jupyter notebook
-================================
+    <div class="sphx-glr-thumbnails">
+        <div class="sphx-glr-thumbcontainer"
+            tooltip="Launch GUI in a Jupyter notebook">
```

Comment:
Why is the tooltip text different from the title? Something about it isn't intuitive to me.